vagrant/bootstrap.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 apt-get update
+apt-get dist-upgrade -y
```

Comment:
NAK. Just start from a modern Ubuntu instead. Upgrading Ubuntu is always full of issues.

Reply:
You are exactly correct. Updating packages was what I tried initially (it didn't work...), then switched to `"ubuntu/focal64"` in this PR.

Reply:
Done